GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -711,12 +722,12 @@ protected override void DisposeCustomResources()
 
         private sealed class ItemTpComparer : IComparer<string>
```

Comment:
Do you know what this class's name means?

Reply:
I have no idea.